web/modules/custom/weather/templates/weather.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="weather-class">
+  {{ temp }}
+</div>
```

Comment:
Such a simple template could be achieved by #container render element
But it is fine if you planning to extend this in future.